aospy/calc.py

```diff
@@ -767,3 +767,19 @@ def load(self, dtype_out_time, dtype_out_vert=False, region=False,
         if plot_units:
             data = self.var.to_plot_units(data, dtype_vert=dtype_out_vert)
         return data
+
+    def addAttrs(self, data):
+        if (isinstance(data, xr.DataArray)):
```

Comment:
Remove the outer set of parentheses (e.g. only `isinstance(data, xr.DataArray)` is needed).